src/vs/workbench/services/part/common/partService.ts

```diff
@@ -129,6 +129,16 @@ export interface IPartService {
 	 */
 	toggleZenMode(): void;
 
+	/**
+	 * Returns whether the centered layout is active.
+	 */
+	isCenteredLayoutActive(): boolean;
```

Comment:
I would rename this to `isLayoutCentered`
Notice how we do not use "active" for similar methods in the same service

Reply:
done.